documentation/tutorials/StateAssimilation/TreeRingSDA.Rmd

```diff
@@ -80,7 +80,7 @@ Perform a site-level SIPNET run using the following settings
   </spin.up>
   <forecast.time.step>1</forecast.time.step>
   <start.date>1980/01/01</start.date>
-  <end.date>2014/12/31</end.date>
+  <end.date>2016/12/31</end.date>
```

Comment:
I can retry it, but this morning while I was trying it through VM it didn't allow